src/uct/ugni/base/ugni_device.c

```diff
@@ -389,26 +372,58 @@ ucs_status_t uct_ugni_iface_get_dev_address(uct_iface_t *tl_iface, uct_device_ad
     return UCS_OK;
 }
 
+static int uct_ugni_next_power_of_two_inclusive (int value)
```

Comment:
@yosefe should it go to ucs as an utility ?

Reply:
up to @hjelmn